buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -756,7 +756,11 @@ class BeamModulePlugin implements Plugin<Project> {
       project.checkstyle { toolVersion = "8.7" }
 
       // Configures javadoc plugin and ensure check runs javadoc.
-      project.tasks.withType(Javadoc) { options.encoding = 'UTF-8' }
+      project.tasks.withType(Javadoc) {
+        options.encoding = 'UTF-8'
+        options.addStringOption('Xdoclint:all', '-quiet')
+        options.addStringOption('Xdoclint:-missing', '-quiet')
```

Comment:
The docs say that these options could be combined as: `'Xdoclint:all,-missing'`

ref: https://docs.oracle.com/javase/8/docs/technotes/tools/unix/javadoc.html

Reply:
Yes.

Actually a

`options.addBooleanOption('Xdoclint:-missing', true)`

would be sufficient as 'on' is the default anyway. But I decided to be consistent with the already existing configuration [1]

Of course, we might change it to the above. But then I would recommend to also do it on javadoc aggregation task. wdyt?

[1] https://github.com/apache/beam/blob/master/sdks/java/javadoc/build.gradle#L77-L78

Reply:
I think the cleanest is to put a single `options.addBooleanOption('Xdoclint:-missing', true)` line here, and to remove the doclint options from the aggregation task. The doclint option setting here should also apply to the aggregation task.